src/components/controls/mouse-controls.js

```diff
@@ -0,0 +1,107 @@
+var registerControls = require('../../core/controls').registerControls;
+var THREE = require('../../lib/three');
+var utils = require('../../utils');
+
+var warn = utils.debug('components:mouse-controls:warn');
+
+/**
+ * Mouse-controls component.
+ *
+ * Updates rotation when dragging the mouse.
+ */
+module.exports.Component = registerControls('mouse-controls', {
```

Comment:
`look-controls` is being updated to listen to drag on window rather than canvas: https://github.com/aframevr/aframe/pull/1474